### PR TITLE
Updated Omnikit readme

### PR DESCRIPTION
**Pull request by Lytrix**
Updated the readme with returning questions asked on Slack:
- How to contribute
- How to setup things including applying for a developer account using the Loop docs.
- Screenshots of the app

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,26 +2,96 @@
 
 [![Join the chat at https://gitter.im/ps2/rileylink](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/ps2/rileylink?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) [![Build Status](https://travis-ci.org/ps2/rileylink_ios.svg?branch=master)](https://travis-ci.org/ps2/rileylink_ios)
 
-The RileyLink iOS app connects to a [RileyLink](https://github.com/ps2/rileylink) via Bluetooth Low Energy (BLE, or Bluetooth Smart) and uploads CGM and pump data to a Nightscout instance via the REST API. The Nightscout web page is also displayed in the App.
+The main purpose of the RileyLink iOS App is for developpers to test and build the device commands. The RileyLink iOS app connects to a [RileyLink](https://github.com/ps2/rileylink) via Bluetooth Low Energy (BLE, or Bluetooth Smart)
```
Comment (ps2) [inline README.md]: developpers -> developers

File: README.md [hunk 2]
```diff
@@ -2,26 +2,96 @@
...
 
-### Getting Started
+## Features
+* Communicate with specific Medtronic Pumps dating before 2012. Read the [Loop documentation](https://loopkit.github.io/loopdocs/setup/requirements/pump/) for more information about which pumps are working.
+* Communicate with the OmniPod. 
+* Uploads CGM and pump data to a Nightscout instance via the REST API. The Nightscout web page is also displayed in the App.
```
Comment (ps2) [inline README.md]: Remove upload and NS info; current app isn't doing that.

File: README.md [hunk 3]
```diff
@@ -2,26 +2,96 @@
...
+### How to Use
+
+To use the Rileylink, it has to connect to a Rileylink device first. If you don't see your RileyLink, close any app that uses the Rileylink or reboot your phone. 
+
+![turn on rileylink in command section](Images/rileylink_ios_setup.PNG)
+
+## Setup an Omnipod
+
+Click on *Setup POD*.
+Follow the same steps as with the PDM, but only push once to insert canula, else the Pod wil fail.
```
Comment (ps2) [inline README.md]: :) should have this bug fixed soon.

**Comment by Lytrix** [comment]
@ps2 could you try restarting the travis build? it had a timeout apparently.

**Pull request merged**